.github/workflows/mixin.yml

```diff
@@ -0,0 +1,32 @@
+name: mixin
+on:
+  push:
+    paths:
+    - 'docs/mixin/alerts/*.yml'
+  pull_request:
+    paths:
+    - 'docs/mixin/alerts/*.yml'
```

Comment:
This is reducing load on CI and running tests only when alerts are changed in PR.